Ex1/primulExercitiu.js

```diff
@@ -0,0 +1,8 @@
+function nrPare(max){
```

Comment:
```suggestion
function returneazaListaCuNrPare(max){
```

Ideal ar fi ca aceasta functie a returneze o noua lista cu numerele pare, iar console.log() sa fie in afara functiei.

Reply:
De asemenea, numele functiei trebuie sa fie un nume destul de citibil incat sa ne dam seama ce face doar din numele ei.